File: day_18_ruby_basics/fizzBuzz.rb
puts "Please enter number 1:\n"
num1 = gets.chomp.to_i
puts "And number 2:\n"
num2 = gets.chomp.to_i

# for n in 1..100
#     if (n % num1==0) && (n % num2==0) puts "FIZZBUZZ\n"
#     elsif (n % num1==0) puts "FiZz\n"
#     elsif (n % num2==0) puts "BuZz\n"
#     else puts "#{n}\n"
# end


for n in 1..100
    str = ""
    str+= "Fizz" if n % num1 == 0
    str+= "Buzz" if n % num2 == 0

    if str.empty?
        puts n
    else
        puts str
    end
end

File: day_2_JS_Basics/fizzbuzz.js
const args = process.argv.slice(2);

const a = +(args[0])
const b = +(args[1])
console.log(a, b);
console.log(a+b);

for (var count=1; count <= 100; count++){
     if (count % a === 0 && count % b === 0)
        console.log("FIZZBUZZ:", count)
     else if (count % a === 0) 
        console.log("FIZZ:", count)
        
     else if (count % b === 0) 
        console.log("BUZZ:", count)
        
     else 
        console.log(count);
}